data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,224 @@ mission "Remnant: Visit to Paradise 1"
+	description "Help the Remnant experiment on the Shattered Light."
+	source "Viminal"
+	destination "Viminal"
+	waypoint "Coluber"
+	to offer
+		has "Remnant: Shattered Light 2: done"
+	to complete
+		has "Remnant: Shattered Light 3: done"
+
+
+
+mission "Remnant: Shattered Light 4"
+	name "Shattered Light"
+	description "Escort the wreck back to Viminal."
+	landing
+	source "Far Monad"
+	destination "Viminal"
+	to offer
+		has "Remnant: Shattered Light 3: done"
+	on offer
+		fail "Remnant: Shattered Light 3b"
+		conversation
+			`Eventually you find a good vantage point several kilometers away where you can watch the approaching ship. The Shattered Light falls through the sky with deceptive slowness, made all the worse by the fact that you can see clouds and sky passing through it. As it nears the surface you see a ripple in the air explode out from it, followed by a concussive shockwave as the Shattered Light suddenly becomes very solid. Its engines flare to life with a roar that rattles your bones and the ship lurches to a stop in mid-air, before crunching down into the forest.`
+			`	The dust and debris settles within a few minutes, and soon you and the <ship> are skimming over the vegetation. The approach gives you ample time to appreciate just how massive the Shattered Light is. Literally everything that one could need to lay the foundations for a new colony, packed into a single ship. You note large gashes in the hull, along with massive sensor arrays and a shattered dome on top. Based on the mismatch of parts, this ship has been substantially modified from its original hull, and survived untold damage.`
+			`	As you slow to a hover alongside, Ruach appears on your viewscreen. "Well met, Captain! Our theory worked! We managed to snap the Shattered Light back into the primary weft, and are now onboard. The systems are dubious, though, so we dare not shut anything down lest we cannot restart it. We would appreciate your assistance in escorting us back to Viminal."`
+				accept
+	npc accompany
+		government "Remnant (Research)"
+		personality launching timid escort
+		planet "Far Monad"
+		ship "Dromedary Wreck" "Shattered Light"
+	npc evade
+		government "Korath"
+		personality entering plunders harvests
+		system Lucina
+		fleet "Korath Ember Waste Raid" 1
+	on complete
+		payment 250000
+		clear "Remnant: Shattered Light 3b: failed"
+		set "Remnant: Shattered Light 3b: done"
+		conversation
+			`	The Shattered Light is so big that traffic control directs the two of you to a large valley next to the spaceport, where you discover that a section of chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of freight, this area looks like it is predominantly used by bulk freight shipments and little else. Now, however, as the Shattered Light slowly moves in, a flock of Pelicans scatter in front of it looking like nothing more than a flock of plovers in front of an approaching camel.`
```

Comment:
```suggestion
			`	The Shattered Light is so massive that traffic control directs your ships to a large valley next to the spaceport, where you discover that a section of the chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of freight, this area looks like it is predominantly used by bulk freight shipments and little else. Now, however, as the Shattered Light slowly moves in, a flock of Pelicans scatter in front of it looking like nothing more than a flock of plovers in front of an approaching camel.`
```

Reply:
your ships doesn't quite sound right. maybe just `directs you to a large valley`? After all, the SL is not one of your ships, and there's no requirement the player have more than one.